contrib/fhs-compliant-script/gitea

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin
+
+########################################################################
+# This script some defaults for gitea to run in a FHS compliant manner #
+########################################################################
+
+# It assumes that you place this script as gitea in /usr/bin
+#
+# And place the original in /usr/lib/gitea with working files in /var/lib/gitea
+# and main configuration in /etc/gitea/app.ini
+GITEA="/usr/lib/gitea/gitea"
+WORK_DIR="/var/lib/gitea"
+APP_INI="/etc/gitea/app.ini"
+
+# Provide FHS compliant defaults to 
+GITEA_WORK_DIR="${GITEA_WORK_DIR:-$WORK_DIR}" GITEA_CUSTOM_PATH="${GITEA_CUSTOM_PATH:-$APP_INI}" "$GITEA" "$@"
```

Comment:
Should ```GITEA_CUSTOM_PATH``` be ```GITEA_CUSTOM```? And if so, it seems this would set it to ```/etc/gitea/app.ini```

Should ```GITEA_CUSTOM_PATH``` be ```$WORK_DIR/custom```  (which I think happens by default if you don't set it)?

So:

```
CUSTOM="$WORK_DIR/custom"

GITEA_WORK_DIR="${GITEA_WORK_DIR:-$WORK_DIR}" GITEA_CUSTOM="${GITEA_CUSTOM:-$CUSTOM}" "$GITEA" "$@" -c $APP_INI
```
 Or something like that?

Reply:
yup you're right.

Reply:
Hmm... let me look again at this.